lib/logging/addon/components/logging/target-embedded/component.js

```diff
@@ -1,4 +1,24 @@
 import Component from '@ember/component';
 import es from 'logging/mixins/target-elasticsearch';
+import { get, set } from '@ember/object';
 
-export default Component.extend(es, {});
+export default Component.extend(es, {
+
+  init(...args) {
```

Comment:
Init doesn't have to take `...args`, Ember convention is `this._super(...arguments)`